staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -137,6 +137,7 @@ func (f *defaultPortForwarder) ForwardPorts(method string, url *url.URL, opts Po
 		return err
 	}
 	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport}, method, url)
```

Comment:
we may have to validate to not go over the limit of streams that we have https://github.com/kubernetes/kubernetes/blob/139fa6962026d1013d1d67e3f9f31d11da290199/staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go#L372-L379

Reply:
resolving as outdated